agent_runtime_profile/CLAUDE.drama.md

```
@@ -139,8 +150,10 @@
 
 ## 项目目录结构
 
+> 下面的目录树仅为说明用途，agent session 的 cwd 已经在项目根，**所有工具参数请使用相对 cwd 的路径**（如 `source/`、`scripts/`），不可带 `projects/{项目名}/` 前缀。
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)